front_end_project/src/pages/email_newsletter_forms/remove.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from 'react';
```

Comment:
Just an FYI, you would not need to import react on every component. Since v17 React. Refer to [this article](https://stackoverflow.com/questions/71725865/do-i-need-to-import-react-in-every-file) for further reading. But this still doesn't affect performance or any metric.

Reply:
Thanks for the tip, appreciate it!